readme.md

```diff
@@ -327,6 +327,7 @@ Thanks for contributing! 🦋🙌
 - [](# "latest-tag-button") [Adds a link to the latest version tag on directory listings and files.](https://user-images.githubusercontent.com/44045911/155496122-6267c45d-21d4-45c9-adf7-94c9e41cc652.png)
 - [](# "convert-release-to-draft") [Adds a button to convert a release to draft.](https://user-images.githubusercontent.com/46634000/139236979-44533bfd-5c17-457d-bdc1-f9ec395f6a3a.png)
 - [](# "link-to-changelog-file") [Adds a button to view the changelog file from the releases page.](https://user-images.githubusercontent.com/46634000/139236982-a1bce2a2-f3aa-40a9-bca4-8756bc941210.png)
+- [](# "allow-submission-via-ctrl-enter") Adds ability to submit new/edited files via Ctrl+Enter .
```

Comment:
This should also be updated to reflect that we're not adding it, but making it work where GitHub forgot, so it should also be moved to the GitHub Shortcomings section

Reply:
Ok, so I updated the description and moved it to the GitHub Shortcomings section. Thanks.